File: CPP/pointer_string_length.cpp
//Calculate the length of a string without built-in functions, using pointers
#include <iostream>
using namespace std;
int main() {
    char str[] = "bujji";
    char *p = str;
    int length = 0;
    while(*p != '\0') {
        length++;
        p++;
    }
    cout << "Length = " << length << endl;
    return 0;
}

File: Java/pointer_string_length.java
//Calculate the length of a string without built-in functions, using pointers
public class pointer_string_length {
    public static void main(String[] args) {
        String str = "bujji";
        int length = 0;
        char[] chars = str.toCharArray();
        for(char c : chars) {
            length++;
        }
        System.out.println("Length = " + length);
    }
}
